src/main/java/uk/ac/ebi/fairwizard/service/ReportBuilder.java

```diff
@@ -248,18 +247,10 @@ void addFairProcess(Document document) throws DocumentException, IOException {
                                   "One the work plan is implemented, users can re-perform FAIR assessment to measure " +
                                   "the outcome of the FAIRification", PARAGRAPH_FONT));
 
-//    Image processDiagram = Image.getInstance("./src/main/resources/fair_process.jpg");
-//    processDiagram.scaleToFit(PageSize.A4.getWidth() * 0.75f, PageSize.A4.getHeight() * 0.75f);
-//    processDiagram.setAlignment(Element.ALIGN_CENTER);
-//    document.add(processDiagram);
-
-    try (InputStream in = getClass().getResourceAsStream("/fair_process.jpg"))  {
-      Image processDiagram = Image.getInstance(in.readAllBytes());
-      processDiagram.scaleToFit(PageSize.A4.getWidth() * 0.75f, PageSize.A4.getHeight() * 0.75f);
-      processDiagram.setAlignment(Element.ALIGN_CENTER);
-      document.add(processDiagram);
-    }
-
+    Image processDiagram = Image.getInstance("./src/main/resources/fair_process.png");
```

Comment:
Get image path from class/container. Otherwise, once pacakged it will stop working. Check how it was before.

Reply:
same as the first .java file problem